lib/challenge.rb

```diff
@@ -16,21 +21,12 @@ def each
       next_fib = current_fib + previous_fib
     end
   end
-
-  attr_reader :stop
```

Comment:
I just wrote it in this order out of habit (modules, attributes, constructor, and then everything else).